demos/README.md

```diff
@@ -6,4 +6,14 @@ This section is meant to house the documentation, configuration, and code needed
 
 ### SDO-OH-EXF-dev-simple
 
-This demo was created for the THINK 2021 conference and begins with the Open Horizon all-in-one Management Hub and Agent installed in a single VM.  It loads the service definition and pattern files for EdgeX Foundry into the Management Hub, and then uses a modified SDO script to first-time boot the simulated Agent, the Agent being automatically configured, and then EdgeX Foundry being automatically deployed by the Agent.  The EdgeX Foundry instance is configured with a Random Number Device so we can demonstrate the platform functioning immediately after installation.
\ No newline at end of file
+This demo was created for the THINK 2021 conference and begins with the Open Horizon all-in-one Management Hub and Agent installed in a single VM.  It loads the service definition and pattern files for EdgeX Foundry into the Management Hub, and then uses a modified SDO script to first-time boot the simulated Agent, the Agent being automatically configured, and then EdgeX Foundry being automatically deployed by the Agent.  The EdgeX Foundry instance is configured with a Random Number Device so we can demonstrate the platform functioning immediately after installation.
```

Comment:
consider splitting up long sentence: "It loads the service definition and pattern files for EdgeX Foundry into the Management Hub, and then uses a modified SDO script to first-time boot the simulated Agent, the Agent being automatically configured, and then EdgeX Foundry being automatically deployed by the Agent."

Reply:
This was not modified from the original, but I agree it can be cleaned up.